sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -2513,6 +2573,12 @@ public partial class CaptionResult
         public string Highlights { get { throw null; } }
         public string Text { get { throw null; } }
     }
+    public partial class DocumentKeysOrIds
```

Comment:
This class name isn't super helpful.  Shouldn't this be something more like `ResetDocumentOptions`?